pandas/tests/sparse/test_frame.py

```diff
@@ -1118,6 +1120,58 @@ def test_isnotnull(self):
         tm.assert_frame_equal(res.to_dense(), exp)
 
 
+@pytest.fixture(params=['bsr', 'coo', 'csc', 'csr', 'dia', 'dok', 'lil'])
+def spmatrix(request):
+    tm._skip_if_no_scipy()
+    from scipy import sparse
+    yield getattr(sparse, request.param + '_matrix')
```

Comment:
you don't use ``yield`` here, just return the result. If this *were* a yield fixture, then you would use ``@pytest.yield_fixture``, but no reason to use one as you don't need to do any cleanup actions.